### ISSUE TITLE
Screen readers cannot announce Message in chat

### ISSUE THREAD

**Issue by Hoang-Long2012** [UnigramDev/Unigram#3284]
Hello.
Currently I'm using NVDA screen reader with Unigram 12.7.1 X64.
Recently my NVDA suddenly stopped reading messages in chats when they had focus. In previous versions of Unigram, NVDA was still able to read messages normally when they were focused.

**Comment by amirmahdifard** [comment]
i'm using Unigram 12.7.4 X64, and there's no problem with nvda.

**Comment by Hoang-Long2012** [comment]
Looks like the error was just due to my configuration, now this problem has been resolved for me, thanks.
